packages/client/components/ReflectionGroup/RemoteReflection.tsx

```diff
@@ -181,12 +182,14 @@ const RemoteReflection = (props: Props) => {
       },
       remoteDrag?.isSpotlight
         ? Times.REFLECTION_SPOTLIGHT_DRAG_STALE_TIMEOUT
+        : localPhase.phaseType === VOTE
```

Comment:
Does this also happen in normal retros? Can you talk me through how this happens? I'm wondering if modifying the `RemoteReflection` component just for the demo is the correct thing to do.

Reply:
@Dschoordsch  It happens at the moment when going from the Group stage to the Vote stage, https://github.com/ParabolInc/parabol/issues/5975 
The participant's RemoteReflection is still in the process of dragging & dropping(not yet), but the meeting facilitator clicks the "Next" button, so it appears on the Vote page that the RemoteReflection still hangs, and will only return to its original position after the animation delay `Times.REFLECTION_DRAG_STALE_TIMEOUT` here, so I set it directly to 0 so it right back to the original position.